src/components/add-review/AddReview.style.ts

```diff
@@ -0,0 +1,61 @@
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
```

Comment:
Here too, you could use styled-components if you'd want. These `StyleSheets` are often used for a proof of concept, but not for larger apps